packages/notifications/push/amplify_push_notifications/android/src/main/kotlin/com/amazonaws/amplify/amplify_push_notifications/PushNotificationConstants.kt

```diff
@@ -0,0 +1,19 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.amazonaws.amplify.amplify_push_notifications
+
+class PushNotificationConstants {
```

Comment:
```suggestion
class PushNotificationPluginConstants {
```
It's too confusing that the constants from the android utils is almost identically named